prompts/ai-coding-assistants/add-module-dependecy.md

```diff
@@ -0,0 +1,346 @@
+---
```

Comment:
not sure that this prompt is necessary, i didnt experience any issues with dependency handling using the AI assistant

Reply:
The more guidance we have for AI the better. Using this requires less input and guarantees a higher success rate.